sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -346,6 +358,15 @@ def send_messages(self, message, **kwargs):
                 :caption: Send message.
 
         """
+        if isinstance(message, list):
```

Comment:
this pattern is seeming rather consistent as I read down the PR.  Wondering if we can pull it off as a whole into a helper function in utils?